walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,42 @@
 
 package com.google.sps;
 
-import java.util.Collection;
+import java.sql.Time;
+import java.util.*;
```

Comment:
Please don't use wildcard imports.
https://engdoc.corp.google.com/eng/doc/devguide/java/style/index.md?cl=head#s3.3-import-statements

Reply:
Unfortunately I cannot access this doc.
But I replace with specific import

Reply:
Also to my defense I think it was not me but IntelliJ :D

Reply:
:D